### PR TITLE
Updated News page

### PR DESCRIPTION
**Pull request by Dejimmy**
@fiestared I updated the news page, could you check it out and merge if OK?

### CODE REVIEW

File: src/yaml/ja.yml [hunk 1]
```diff
@@ -408,6 +408,60 @@ openb:
   btn_text: 詳細はこちら
   url: https://alis.to/ALIS-official/articles/3qQXJLNJA4NM
 news:
+  _08142020:
+    category: NEWS
+    date: Aug 14, 2020
+    header: 坂井教授著の「メカニズムデザインで勝つ」に掲載
+    body: 慶應大学教授・坂井教授&オークション･ラボ執筆の「メカニズデザインで勝つ」でALISのMajority Judgementsの実験が紹介されました。
+    url: https://www.amazon.co.jp/dp/B08FDBCDLJ/ref=dp-kindle-redirect?_encoding=UTF8&btkr=1
```
Comment (fiestared) [inline src/yaml/ja.yml:416]: I think you might better change this url to the one below. It worked well, but the parameter doesn't need to be put on.
https://www.amazon.co.jp/dp/B08FDBCDLJ/

**Comment by fiestared** [comment]
@Dejimmy I just left one comment, so please check and modify it, then commit the code again. Everything else looks good to me!

**Comment by Dejimmy** [comment]
@fiestared I changed the link to the one you suggested.

**Comment by fiestared** [comment]
@Dejimmy I'm so sorry for replying late. LGTM!

**Pull request merged**